kani-driver/src/project.rs

```diff
@@ -177,6 +177,13 @@ impl Artifact {
 /// be collected from the project.
 pub fn cargo_project(session: &mut KaniSession, keep_going: bool) -> Result<Project> {
     let outputs = session.cargo_build(keep_going)?;
+    if session.args.no_codegen {
+        info_operation(
+            "info",
+            "Skipping codegen. Rerun without `--no-codegen` to perform codegen.",
```

Comment:
Nit:
```suggestion
            "Compilation succeeded up until codegen. Skipping codegen because of `--no-codegen` option. Rerun without `--no-codegen` to perform codegen.",
```

The point being to give some indicator of success.